build.gradle.kts

```diff
@@ -20,12 +22,24 @@ plugins {
     `java-library`
 
     id("org.jetbrains.kotlin.jvm") version Version.kotlin apply(false)
+    id("io.github.gradle-nexus.publish-plugin") version Version.nexusPublishPlugin
 }
 
-subprojects {
-    group = "io.provenance.scope"
-    version = project.property("version")?.takeIf { it != "unspecified" } ?: "1.0-SNAPSHOT"
+group = "io.provenance.scope"
```

Comment:
according to the plugin docs, this should be at the root-level, I think this is equivalent to having it in subprojects though 🤞